config/locales/en.yml

```diff
@@ -99,42 +99,48 @@ en:
     upcoming_bank_holidays: Upcoming bank holidays
   continue: Continue
   cookies:
-    additional: Most services we link to are run by different government departments, for example DWP’s Universal Credit online, DVLA’s vehicle tax service, or HMRC’s webchat. These services may set additional cookies and, if so, will have their own cookie policy and banner linking to it.
     always_on: They always need to be on.
-    clicks: what you click on while you're visiting the site
+    clicks: what you click on while you're visiting these sites
     confirmation_message: Government services may set additional cookies and, if so, will have their own cookie policy and banner.
     confirmation_previous_referrer_link: Go back to the page you were looking at
     confirmation_title: Your cookie settings were saved
+    explaination:
+      body: This page has a brief explanation of each type of cookie we use. If you want more details,
+      link: read our detailed cookie information
```

Comment:
Rather than split these into two keys, I'd suggest having one HTML field instead, like in https://github.com/alphagov/frontend/blob/d72c1dff5eb83bb3d1afaa1149a49fab3644200a/config/locales/en.yml#L8. That makes our approach consistent with what's in Frontend already, and should make it easier to accommodate things like RTL languages in future, as they wouldn't be locked into the markup we've hardcoded into the view.

Reply:
We seem to have inconsistency on this based on which repo we're in. Other repos have the links separated rather than a large HTML string in the translation, which is much cleaner. I have done as you've stated in this case, but I do not feel they're particularly sensible as a solution.